index.md

```diff
@@ -99,28 +96,31 @@ like the default scheduler, in the cluster: [Co-existing with other K8s schedule
 | 1.13.x | 0.8.0 | 0.10.0 |
 | 1.14.x | 0.8.0 | 0.10.0 |
 | 1.15.x | 0.8.0 | 0.10.0 |
-| 1.16.x | 0.10.0 | ? |
-| 1.17.x | 0.10.0 | ? |
-| 1.18.x | 0.10.0 | ? |
+| 1.16.x | 0.10.0 | 0.11.0 |
+| 1.17.x | 0.10.0 | 0.11.0 |
+| 1.18.x | 0.10.0 | 0.11.0 |
+| 1.19.x | 0.11.0 | - |
+| 1.20.x | 0.12.1 | - |
+| 1.21.x | 0.12.1 | - |
+| 1.22.x | 0.12.2 (upcoming) | - |
+| 1.23.x | 0.12.2 (upcoming) | - |
```

Comment:
we should never point to unreleased versions

Reply:
fixed